subprojects/ide-native/src/main/java/org/gradle/ide/xcode/plugins/XcodePlugin.java

```diff
@@ -312,7 +312,7 @@ public void execute(final Project project) {
                 xcodeProject.getGroups().getSources().from(sources);
 
                 FileCollection headers = component.getHeaderFiles();
-                xcodeProject.getGroups().getHeaders().from(headers);
+                xcodeProject.getGroups().getHeaders().from(headers.getAsFileTree());
```

Comment:
The fix should go in the implementation of `getHeaderFiles()`. It should return all of the header files.

Reply:
Good point, let me fix those.